examples/listbox_sort_stringlist/main.rs

```diff
@@ -0,0 +1,147 @@
+// Variation on ListBox model example to show use of StringList and StringSorter.
+
+// TODO: add a dialog window to allow adding rows.
+// use plain gtk::Window for this per current practice.
+
+use gtk::{
+    glib::{self, clone},
+    prelude::*,
+};
+
+fn main() -> glib::ExitCode {
+    let application = gtk::Application::builder()
+        .application_id("com.github.gtk-rs.examples.listbox-sorted-StringList")
+        .build();
+
+    application.connect_activate(build_ui);
+
+    application.run()
+}
+
+fn build_ui(application: &gtk::Application) {
+    let window = gtk::ApplicationWindow::builder()
+        .default_width(320)
+        .default_height(480)
+        .application(application)
+        .title("Sorted StringList")
+        .build();
+
+    let vbox = gtk::Box::new(gtk::Orientation::Vertical, 5);
+
+    // Create a StringSorter with a property expression to sort
+    // StringObjects in a StringList.  StringObject has a "string" property.
+    let expression = gtk::PropertyExpression::new(
+        gtk::StringObject::static_type(),
+        None::<gtk::Expression>,
+        "string",
+    );
+    let sorter = gtk::StringSorter::new(Some(expression));
+    sorter.set_ignore_case(true);
+
+    // Create our list store as a StringList and populate with some strings.
+    let model = gtk::StringList::new(&["zoo", "abba", "donkey", "sunrise", "river", "phoenix"]);
+
+    // Create a sort model and bind it to the ListStore and the sorter.
+    let sort_model = gtk::SortListModel::new(Some(model.clone()), Some(sorter.clone()));
```

Comment:
```suggestion
    let sort_model = gtk::SortListModel::new(Some(model.clone()), Some(sorter));
```
you don't use it later on, so the clone is not needed